src/tools/sfdk/dbus.cpp

```diff
@@ -82,6 +87,13 @@ class ConfiguredDeviceAdaptor : public AbstractAdaptor
     }
 
 public slots:
+    bool isEmulator(const QDBusMessage &message)
```

Comment:
I don't think this is actually used anywhere

Reply:
never mind, of course it's called from dbus.

Reply:
Is it? I didn't find it quickly so I thought I actually forgot to utilize it on sdk-setup side - now it would be nice to know if you found it somewhere :)

Reply:
No I actually didn't.

Reply:
Yeah and I forgot to fix it before merging. Anyway, noted down and I will fix it while working on the autotests.